cmd/sentinel/cmd/sentinel.go

```diff
@@ -1833,24 +1834,24 @@ func (s *Sentinel) leaderInfo() (bool, uint) {
 	return s.leader, s.leadershipCount
 }
 
-func (s *Sentinel) clusterSentinelCheck(pctx context.Context) {
+func (s *Sentinel) clusterSentinelCheck(pctx context.Context) bool {
```

Comment:
what is the meaning/interpretation of this bool return value? 

a comment would be helpful

and/or, make it explicit in the code by doing something like `shouldContinueWaiting := s.clusterSentinelCheck(ctx)` above

Reply:
Will add a comment 👍